Calling instance method from getter in VueSingleFileComponent causes error: Cannot find module

I've a vue 3 project with typescript and I'm trying to get ts-transformer-keys work.

My configuration:

tsconfig.json:
```json
"plugins": [
      { "transform": "ts-transformer-keys/transformer" }
]
```

vue.config.js
```js
chainWebpack: config => {
    ['ts', 'tsx'].forEach(rule => {
      config.module
        .rule(rule)
        .use('ts-loader')
        .tap(options => Object.assign(options, {
          compiler: 'ttypescript',
          transpileOnly: false
        }))
    })
  }
```

I could locate the issue in the following problem:
If I call an instance method from a getter I got an error 
> Error: Cannot find module 'C:\path\src\App.vue.ts'

Without `ts-transformer-keys` this works fine.

For Example:
App.vue:

```vue
<template>
  <p>{{ message }}</p>
  <p>{{ messageAlias }}</p>
  <p>{{ upperCaseMessage }}</p>
  <p>{{ messageToUpperCase }}</p>
</template>

<script lang="ts">
import { Vue } from 'vue-class-component'
export default class App extends Vue {
  message = 'hello World'

  get messageAlias() {
    return this.message
  }

  get upperCaseMessage() {
    // return this.message.toUpperCase() // this will work
    return this.messageToUpperCase() // this won't
  }

  messageToUpperCase() {
    return this.message.toUpperCase()
  }
}
</script>
```

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -1,0 +1,18 @@
+name: test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm test

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,0 @@
-language: node_js
-node_js:
-  - '10'

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # ts-transformer-keys
 A TypeScript custom transformer which enables to obtain keys of given type.
 
-[![Build Status][travis-image]][travis-url]
+![Build Status](https://github.com/kimamula/ts-transformer-keys/workflows/test/badge.svg)
 [![NPM version][npm-image]][npm-url]
 [![Downloads](https://img.shields.io/npm/dm/ts-transformer-keys.svg)](https://www.npmjs.com/package/ts-transformer-keys)
 
@@ -183,7 +183,5 @@ class MyClass<T extends object> {
 
 MIT
 
-[travis-image]:https://travis-ci.org/kimamula/ts-transformer-keys.svg?branch=master
-[travis-url]:https://travis-ci.org/kimamula/ts-transformer-keys
 [npm-image]:https://img.shields.io/npm/v/ts-transformer-keys.svg?style=flat
 [npm-url]:https://npmjs.org/package/ts-transformer-keys

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
   "devDependencies": {
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.0.13",
-    "mocha": "^8.0.1",
+    "mocha": "^8.2.1",
     "typescript": "^3.9.5"
   },
   "peerDependencies": {

--- a/transformer.ts
+++ b/transformer.ts
@@ -51,13 +51,17 @@ function isKeysCallExpression(node: ts.Node, typeChecker: ts.TypeChecker): node 
   if (!ts.isCallExpression(node)) {
     return false;
   }
-  const signature = typeChecker.getResolvedSignature(node);
-  if (typeof signature === 'undefined') {
+  const declaration = typeChecker.getResolvedSignature(node)?.declaration;
+  if (!declaration || ts.isJSDocSignature(declaration) || declaration.name?.getText() !== 'keys') {
     return false;
   }
-  const { declaration } = signature;
-  return !!declaration
-    && !ts.isJSDocSignature(declaration)
-    && require.resolve(declaration.getSourceFile().fileName) === indexTs
-    && declaration.name?.getText() === 'keys';
+  try {
+    // require.resolve is required to resolve symlink.
+    // https://github.com/kimamula/ts-transformer-keys/issues/4#issuecomment-643734716
+    return require.resolve(declaration.getSourceFile().fileName) === indexTs;
+  } catch {
+    // declaration.getSourceFile().fileName may not be in Node.js require stack and require.resolve may result in an error.
+    // https://github.com/kimamula/ts-transformer-keys/issues/47
+    return false;
+  }
 }

